packages/helpermodules/update_config.py

```diff
@@ -1821,3 +1822,12 @@ def upgrade_datastore_62(self) -> None:
             "Fehlerfall die Ladung gestoppt. Du kannst die maximale Leistung im Fehlerfall für jeden Zähler"
             " unter Einstellungen -> Konfiguration -> Lastmanagement anpassen.", MessageType.WARNING)
         self.__update_topic("openWB/system/datastore_version", 63)
+
+    def upgrade_datastore_63(self) -> None:
+        def upgrade(topic: str, payload) -> Optional[dict]:
+            if re.search("openWB/bat/[0-9]+/get/power", topic) is not None:
+                index = get_index(topic)
+                if f"openWB/bat/{index}/get/power_limit_controllable" not in self.all_received_topics.keys():
+                    return {f"openWB/bat/{index}/get/power_limit_controllable": False}
```

Comment:
Das mag jetzt so passen (abgesehen von dem MQTT Speichermodul). Wenn aber jemand ein Update ausführt auf einen Versionsstand, in dem ein angelegter Speicher die Funktionalität bereits unterstützt, ist `False` hier falsch.
Es sollte also aktiv geprüft werden, ob sich die vorhandene Speicherkomponente steuern lässt.

Reply:
Die Regelung prüft, ob sich der Speicher steuern lässt und setzt das Topic dann auf True.
Der Speicher kann ja auch schon angelegt sein und die Speicher-Steuerung erst später implementiert werden.